marketplaceItems/questionApp/appResources/appData/question_app.js

```diff
@@ -38,50 +38,62 @@ Tablet, Users, Vec3, Window */
     avatar's head and creates a question mark entity there */
-    var QUESTION_MARK_Y_OFFSET_RATIO_FROM_HEAD_TOP = 0.3;
-    var QUESTION_MARK_Y_OFFSET_RATIO_FROM_HEAD = 0.4;
-    var QUESTION_MARK_Y_OFFSET_RATIO_FROM_HIPS = 0.6;
-    var GROWTH_INTERVAL_MS = 1000; // takes 4 min 38 sec to reach height of 0.8 from 0.2 at ratio:1.005, interval: 1000
+    var QUESTION_MARK_Y_OFFSET_RATIO_FROM_HEAD_TOP = 0.1;
+    var QUESTION_MARK_Y_OFFSET_RATIO_FROM_HEAD = 0.28;
+    var QUESTION_MARK_Y_OFFSET_RATIO_FROM_HIPS = 0.55;
+    var GROWTH_INTERVAL_MS = 500; // takes 4 min 38 sec to reach height of 0.8 from 0.2 at ratio:1.005, interval: 1000
     var MAX_HEIGHT_M = 0.8;
-    var GROWTH_RATIO = 1.1;
+    var GROWTH_RATIO = 1.005;
     var QUESTION_MARK_PROPERTY_NAME = "Question App Mark";
-    var QUESTION_MARK_START_DIMENSIONS_M = { x: 0.2, y: 0.2, z: 0.2 };
+    var QUESTION_MARK_START_DIMENSIONS_M = { x: 0.1, y: 0.2, z: 0.02 };
+    var HIFI_CYAN = { red: 0, green: 158, blue: 224 };
+    var HIFI_YELLOW = { red: 255, green: 237, blue: 0 };
+    var HIFI_RED = { red: 255, green: 0, blue: 16 };
+    var HALF = 0.5;
     var questionMark;
     var questionMarkMaterial;
-    var growingInterval;
+    var changingInterval;
+    var parentJointIndex;
+    var offsetRatio;
     var lastDimensions = QUESTION_MARK_START_DIMENSIONS_M;
     function createQuestionMark() {
         if (questionMark) {
             return;
         }
         var avatarHeight = MyAvatar.getHeight();
-        var parentJointIndex = MyAvatar.getJointIndex("HeadTop_End");
-        var offsetRatio = QUESTION_MARK_Y_OFFSET_RATIO_FROM_HEAD_TOP;
-        print("USING HEAD TOP JOINT");
+        parentJointIndex = MyAvatar.getJointIndex("HeadTop_End");
+        offsetRatio = QUESTION_MARK_Y_OFFSET_RATIO_FROM_HEAD_TOP;
         if (parentJointIndex === -1) {
-            print("NOPE>>>USING HEAD JOINT");
             parentJointIndex = MyAvatar.getJointIndex("Head");
             offsetRatio = QUESTION_MARK_Y_OFFSET_RATIO_FROM_HEAD;
         }
         if (parentJointIndex === -1) {
             parentJointIndex = MyAvatar.getJointIndex("Hips");
             offsetRatio = QUESTION_MARK_Y_OFFSET_RATIO_FROM_HIPS;
-            print("ERROR: Falling back to hips joint as head could not be found");
         }
-        var questionMarkLocalYPosition = avatarHeight * offsetRatio;
+        var questionMarkLocalYPosition = avatarHeight * offsetRatio + HALF * QUESTION_MARK_START_DIMENSIONS_M.y;
         var questionMarkProperties = {
             name: QUESTION_MARK_PROPERTY_NAME,
             type: "Model",
-            modelURL: Script.resolvePath("resources/models/sphere-white-emissive.fbx"),
+            modelURL: Script.resolvePath("resources/models/questionMark.fbx"),
             lifetime: 360000,
             parentID: MyAvatar.sessionUUID,
             parentJointIndex: parentJointIndex,
+            // angularVelocity: { x: 0, y: 5, z: 0 },
```

Comment:
necessary? i think the spin could be a useful feature addition. however, if you're not going to implement that, please remove these lines

Reply:
I was going to ask you to look at it but Ryan said we should not have spin, so deleting